pkg/controller/operands/kubevirt.go

```diff
@@ -254,6 +255,49 @@ func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtv1.KubeVirtConfigurati
 	return config, nil
 }
 
+func toKvPermittedHostDevices(permittedDevices *hcov1beta1.PermittedHostDevices) *kubevirtv1.PermittedHostDevices {
```

Comment:
I we just pass through this setting transparently to kubevirt by definition, do we really want to mirror the API type here?

We could also use `kubevirtv1.PermittedHostDevices/MediatedHostDevice` directly, so we just have to bump our kubevirt dependency to add updated documentation/settings from kubevirt instead of having to change code.

If there are ever settings introduced in KubeVirt that we don't want to expose via HCO, we can always replace the kubevirt reference with our own struct as long as the json annotations stay compatible.
Obviously we have to ensure that we don't accidentally release HCO with new options here, but that should be easy to spot in code reviews when we update kubevirt.

Reply:
We thought about it, but we don't want to be affected by other compponent when it comes to the API itself, in order to keep HCO API in HCO control; e.g for documentation an so on.

Reply:
@tiraboschi  ^^^

Reply:
nvm. Makes sense, just wanted to check :+1: